simple_slam:
  frame_id_: "scan"
  angle_min_: 0
  angle_max_: 6.2831853
  angle_increment_: 0.012566371
  scan_time_: 0.1
  range_min_: 0.7
  range_max_: 20
  inf_epsilon_: 1.0
  z_min_: -0.2
  z_max_: 0.0
  map_resolution_: 0.1
  init_map_size_: 400
  map_expand_size_: 200
  occupancy_center_add_: 0.55
  occupancy_center_sub_: 0.45
  occupancy_constant_add_: 1.25
  occupancy_constant_sub_: 0.95
